WebRoot/templates/prefs/Pages.template

```diff
@@ -644,33 +644,6 @@ and the container element is replaced with that control.
 	<$ } $>
 </template>
 
-<!-- NOTE: we have two separate templates for white/black lists for tabbing support -->
```

Comment:
This comment says there were two separate templates to have tabbing support. Can we verify if tabbing works now with our changes?